core/src/test/java/org/pentaho/di/core/xml/XMLHandlerUnitTest.java

```diff
@@ -200,7 +200,7 @@ public void addTagValueDouble() {
   @Test
   public void addTagValueBinary() throws IOException {
     byte[] input = "Test Data".getBytes();
-    String result = "H4sIAAAAAAAAAAtJLS5RcEksSQQAL4PL8QkAAAA=";
+    String result = "H4sIAAAAAAAA/wtJLS5RcEksSQQAL4PL8QkAAAA=";
```

Comment:
What changed here, just an implementation thing in the JDK?

Reply:
Exactly!

Reply:
This unit test is broken in master (at least in wingman) since the merge. The old value seems to work.